schema-migration-helper-service.md

```diff
@@ -0,0 +1,560 @@
+# Schema Migration Helper Guide
+
+## Why have a Schema Migration Helper?
+
+Schema migrations are a general mechanism that Fivetran uses to manage DDL/DML operations on existing destination tables. These migrations are occasionally needed to safely change, update, or repair tables and columns in a customer's destination without requiring the customer to run SQL or reload the connector.
+
+Some frequently used schema migrations have been grouped into more easily implemented 'complex' or 'advanced' schema migration operations. These grouped operations provide standardized abstractions that are advantageous for partners to implement, as they handle common scenarios with built-in best practices for data integrity and history preservation.
+
+> Important: If some migration operations are not implemented, certain Fivetran dashboard features will not work correctly for connections using your connector. For example, customers may be unable to switch sync modes in the dashboard, or bulk schema fixes may fail to apply automatically.
```

Comment:
Instead of a single "For example, ..." , these should all be spelled out, like which ones make which features work, so they will know